diff --git a/arch/mips/mm/c-r4k.c b/arch/mips/mm/c-r4k.c
index 6bacb519..478de865 100644
--- a/arch/mips/mm/c-r4k.c
+++ b/arch/mips/mm/c-r4k.c
@@ -62,7 +62,7 @@ static unsigned long scache_size __read_mostly;
 static unsigned long icache_size, dcache_size, scache_size;
 
 /* RYH */
-static int __attribute_unused__ rac_set = 0;
+static int __attribute__((__unused__)) rac_set = 0;
 extern int par_val;
 extern char cfeBootParms[]; 
 extern unsigned long rac_config0, rac_config1, rac_address_range;
diff --git a/arch/mips/mm/init.c b/arch/mips/mm/init.c
index fc08ddd8..1accea9b 100644
--- a/arch/mips/mm/init.c
+++ b/arch/mips/mm/init.c
@@ -396,9 +396,9 @@ void __init paging_init(void)
 {
 	unsigned long zones_size[] = { [0 ... MAX_NR_ZONES - 1] = 0 };
 	unsigned long max_dma, high, low;
-	unsigned long __attribute_unused__ zholes_size[] =
+	unsigned long __attribute__((__unused__)) zholes_size[] =
 		{ [0 ... MAX_NR_ZONES - 1] = 0 };
-	unsigned long __attribute_unused__ i, j, pfn;
+	unsigned long __attribute__((__unused__)) i, j, pfn;
 
 	pagetable_init();
 
diff --git a/drivers/net/brcmint7038/bcmemac.c b/drivers/net/brcmint7038/bcmemac.c
index c1216f77..7eeb97a2 100644
--- a/drivers/net/brcmint7038/bcmemac.c
+++ b/drivers/net/brcmint7038/bcmemac.c
@@ -2882,7 +2882,7 @@ static int bcmemac_uninit_dev(BcmEnet_devctrl *pDevCtrl)
 {
     Enet_Tx_CB *txCBPtr;
     char proc_name[32];
-    int i, devnum __attribute_unused__ = pDevCtrl->devnum;
+    int i, devnum __attribute__((__unused__)) = pDevCtrl->devnum;
 
     if (pDevCtrl) {
         /* disable DMA */
